scripts/inc.compatibility.sh

```diff
@@ -27,7 +27,8 @@ keyboardlibhack(){ #only on lollipop and higher
 /system/app/LatinIME/lib/$ARCH/libjni_latinimegoogle.so"
     KEYBDLIBS='keybd_lib_filename1="libjni_latinimegoogle.so";
 keybd_lib_filename2="libjni_latinime.so";'
-    KEYBDINSTALLCODE='if ( ! contains "$gapps_list" "keyboardgoogle" ); then
+    # Skip keyboard treatment on CyanogenMod without Google version (CM's LatinIME is broken with Google lib)
```

Comment:
Are you sure that CM's LatinIME is always broken with the Google lib?
We had some reports of some issues sometimes, but for other people it does seem to work :-/